app/routes/object_storage/objects.py

```diff
@@ -110,6 +118,7 @@ def list_objects(namespace_name, bucket_name):
         content_type="application/json",
         response=json.dumps({"objects": objects, "prefixes": prefixes}),
         headers={
+            "etag": str(uuid.uuid4()),
```

Comment:
This route is returning a list of objects. So this information shouldn't be placed here.

After doing the changes described on the upper comment look for the file `app/resources/object_storage/objects.py` and make this change for variable `_objects`.

```python
    _objects = [
        {"name": _object["object_name"], "etag": _object["etag"]}
        for _object in bucket["_objects"]
    ]
```